composer.json

```diff
@@ -15,11 +15,11 @@
         "guzzlehttp/guzzle": "^6.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.2",
-        "codedungeon/phpunit-result-printer": "^0.19.10",
-        "nunomaduro/collision": "^2.0",
+        "phpunit/phpunit": "^8.5",
+        "codedungeon/phpunit-result-printer": "^0.27.0",
+        "nunomaduro/collision": "^3.0",
```

Comment:
we could even go to `^4.1.3`

Reply:
updating this dependency seems to break something within phpunit:

```
> vendor/bin/phpunit
Class "NunoMaduro\Collision\Adapters\Phpunit\Listener" does not exist
Script vendor/bin/phpunit handling the test event returned with error code 2
```

Reply:
I saw that we had to update the php-unit file. I just commited it. 👍

Reply:
sry, i forgot that it's a for from your side ^^

here would be the new config-file:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<phpunit bootstrap="vendor/autoload.php" backupGlobals="false" backupStaticAttributes="false" colors="true"
         verbose="true" convertErrorsToExceptions="true" convertNoticesToExceptions="true"
         convertWarningsToExceptions="true" processIsolation="false" stopOnFailure="false"
         printerClass="NunoMaduro\Collision\Adapters\Phpunit\Printer">
    <testsuites>
        <testsuite name="Towa Test Suite">
            <directory>tests</directory>
        </testsuite>
    </testsuites>
    <filter>
        <whitelist>
            <directory suffix=".php">src/</directory>
        </whitelist>
    </filter>
</phpunit>
```